packages/utils/recursive-parser.js

```diff
@@ -4,38 +4,51 @@ const utils = require("./ast-utils");
 
 module.exports = function recursiveTransform(j, ast, value, action, key) {
 	const node = utils.findRootNodesByName(j, ast, key);
+
+	// get module.exports prop
+	const root = ast
+		.find(j.ObjectExpression)
+		.filter(p => {
+			return (
+				utils.safeTraverse(p, [
+					"parentPath",
+					"value",
+					"left",
+					"object",
+					"name"
+				]) === "module" &&
+				utils.safeTraverse(p, [
+					"parentPath",
+					"value",
+					"left",
+					"property",
+					"name"
+				]) === "exports"
+			);
+		})
+		.filter(p => p.value.properties);
+
 	if (node.size() !== 0) {
-		// push to existing key
-		return ast;
+		// select node with existing key
+		return utils.findRootNodesByName(
+			j, root, key
+		)
+			.forEach(p => {
```

Comment:
What is `p`? Could we choose a better name?

Reply:
But `p` has a different meaning based on where it is used. We have JSDoc that helps us already. At least let's give something to the developers, so when they land to the code they know what that variable is meant for

Reply:
p is alright